### PR TITLE
Adding sphinx-jupyterbook-latex package

### PR DESCRIPTION
**Pull request by AakashGfude**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AakashGfude** [comment]
@bollwyvl @chrisjsewell  @mmcky , this looks good to go.

**Comment by AakashGfude** [comment]
@conda-forge-admin please rerender

**Comment by AakashGfude** [comment]
@conda-forge-admin, please restart ci

**Comment by AakashGfude** [comment]
@bollwyvl @chrisjsewell @mmcky, this looks good to go.

**Comment by mmcky** [comment]
@AakashGfude merge this when you're ready. I have no `merge` rights on this repo.

**Comment by mmcky** [comment]
@AakashGfude does a feedstock repo get setup from this?

**Comment by AakashGfude** [comment]
> @AakashGfude merge this when you're ready. I have no `merge` rights on this repo.

@mmcky I can't merge as well.

File: recipes/sphinx-jupyterbook-latex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (bollwyvl) [inline recipes/sphinx-jupyterbook-latex/meta.yaml:1]: as suggested, we're probably ready to clean out these comments

File: recipes/sphinx-jupyterbook-latex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - pip
+  commands:
+    - pip check
+  imports:
+    - sphinx_jupyterbook_latex
+    - sphinx_jupyterbook_latex.theme
+
+
+about:
+  home: "https://github.com/executablebooks/sphinx-jupyterbook-latex"
```
Comment (bollwyvl) [inline recipes/sphinx-jupyterbook-latex/meta.yaml:44]: none of these quotes are necessary

File: recipes/sphinx-jupyterbook-latex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - sphinx >=3,<4
+    - myst-nb >=0.11,<0.13
+    - sqlalchemy ~=1.3.12
```
Comment (bollwyvl) [inline recipes/sphinx-jupyterbook-latex/meta.yaml:31]: not a huge fan of the tildes: would prefer `>=1.3.12,<1.4`

File: recipes/sphinx-jupyterbook-latex/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "sphinx-jupyterbook-latex" %}
+{% set pip_name = name.replace("-", "_") %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
```
Comment (bollwyvl) [inline recipes/sphinx-jupyterbook-latex/meta.yaml:11]: doesn't need lower

File: recipes/sphinx-jupyterbook-latex/meta.yaml (hunk 1 above)
Comment (AakashGfude) [reply recipes/sphinx-jupyterbook-latex/meta.yaml]: Done in https://github.com/conda-forge/staged-recipes/pull/15234/commits/7b9c4049f152539d813b8c4bf547b77459ba6f8b

File: recipes/sphinx-jupyterbook-latex/meta.yaml (hunk 4 above)
Comment (AakashGfude) [reply recipes/sphinx-jupyterbook-latex/meta.yaml]: Done in https://github.com/conda-forge/staged-recipes/pull/15234/commits/7b9c4049f152539d813b8c4bf547b77459ba6f8b

File: recipes/sphinx-jupyterbook-latex/meta.yaml (hunk 2 above)
Comment (AakashGfude) [reply recipes/sphinx-jupyterbook-latex/meta.yaml]: Done in https://github.com/conda-forge/staged-recipes/pull/15234/commits/7b9c4049f152539d813b8c4bf547b77459ba6f8b

**Comment by AakashGfude** [comment]
Thank you @bollwyvl for your helpful suggestions

File: recipes/sphinx-jupyterbook-latex/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  summary: |
+    Sphinx-jupyterbook-latex: Supporting LaTeX infrastructure for Jupyter Book
+  dev_url: https://github.com/executablebooks/sphinx-jupyterbook-latex
+
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    - AakashGfude
+    - conda-forge/jupyter-book
```
Comment (bollwyvl) [inline recipes/sphinx-jupyterbook-latex/meta.yaml:51]: Almost all are already members of @conda-forge/jupyter-book (for whom i hereby accept maintainership)... @mmcky  can you just drop a comment that you are interested in helping maintain this?

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review.

this is blocking https://github.com/conda-forge/jupyter-book-feedstock/pull/6

Thanks!

**Comment by ocefpaf** [comment]
> @conda-forge/staged-recipes @conda-forge/help-python ready for review.

@bollwyvl you are the best person we have to review jupyter related PRs. Maybe it is time to be part of the staged-recipes team so you can merge these PRs ;-p

**Comment by bollwyvl** [comment]
Thanks! Hopefully this jupyter-book upgrade still be less... Exciting.

File: recipes/sphinx-jupyterbook-latex/meta.yaml (hunk 5 above)
Comment (mmcky) [reply recipes/sphinx-jupyterbook-latex/meta.yaml:51]: @bollwyvl sorry for slow reply. Happy to help maintain this.

**Pull request merged**